opentelemetry-appender-log/src/lib.rs

```diff
@@ -621,7 +621,7 @@ mod any_value {
 
         fn end(self) -> Result<Self::Ok, Self::Error> {
             Ok(Some(AnyValue::Map({
-                let mut variant = HashMap::new();
+                let mut variant = HashMap::default();
```

Comment:
are these related to this PR?

Reply:
these were the related to lint error with box'ing change. I didn't revert this while revering box'ing, as both are similar, and `default` is more explicit.